File: app/src/main/java/com/example/prennote/Splash.java
package com.example.prennote;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import androidx.appcompat.app.AppCompatActivity;

import com.example.prennote.fragments.HomeFragment;

public class Splash extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        Handler handler = new Handler();
        handler.postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent gotohome = new Intent(Splash.this, HomeFragment.class);
                startActivity(gotohome);
                finish();
            }
        }, 5000);
    }
}

File: app/src/main/java/com/example/prennote/Splash.kt
package com.example.prennote

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity
import com.example.prennote.fragments.HomeFragment

class Splash : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        val handler = Handler()
        handler.postDelayed(object : Runnable {
            override fun run() {
                val gotohome = Intent(this@Splash, HomeFragment::class.java)
                startActivity(gotohome)
                finish()
            }
        }, 5000)
    }
}